lib/components/badgeGrid.dart

```diff
@@ -2,26 +2,27 @@ import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:nativemoon/services/badge.dart';
 
-class BadgeGrid{
-
+class BadgeGrid {
   final List<Badge> badges;
-  final List<Image> images;
 
-  BadgeGrid(this.badges, this.images);
+  BadgeGrid(this.badges);
 
-  List<Card> buildGrid() {
+  List<Card> buildGrid(BuildContext context) {
```

Comment:
This change is causing an error on file lib/screens/Home/index.dart line 85.

Reply:
I believe this is fixed now.